pnpm-lock.yaml

```diff
@@ -60,7 +60,7 @@ importers:
   packages/cssnano:
     specifiers:
       autoprefixer: ^10.3.7
-      cssnano-preset-advanced: ^5.2.4
+      cssnano-preset-advanced: ^5.2.5
```

Comment:
Forgot to update this dependency to `workspace` in https://github.com/cssnano/cssnano/commit/a0132086dd3ab9cd7285102debf0e7f9a1de3b92